spark-indexer/src/main/java/org/uniprot/store/spark/indexer/go/evidence/GOEvidenceMapper.java

```diff
@@ -70,16 +69,16 @@ private UniProtKBCrossReference addGoEvidences(
     }
 
     private Map<String, List<Evidence>> getGoEvidenceMap(Iterable<GOEvidence> goEvidences) {
```

Comment:
Can you please create unit test for this code change to make sure that this scenario is covered. it will prevent future changes to break it.